src/utils/showPreviousMessage.js

```diff
@@ -0,0 +1,35 @@
+const UP_KEY_CODE = 38;
+const DOWN_KEY_CODE = 40;
+
+const showPreviousMessage = (keyCode, messageHistory, currentIndex) => {
```

Comment:
Name is not suitable.

Reply:
should I name it `onChatComposerKeydown`

Reply:
@Dilshaad21 Yes